sdks/python/apache_beam/runners/portability/portable_runner.py

```diff
@@ -239,6 +240,7 @@ def send_options_request(max_retries=5):
           num_retries += 1
           if num_retries > max_retries:
             raise e
+          time.sleep(1)
```

Comment:
FYI Included this here to give a bit more time for the channel to come up. Without waiting, it retries immediately and the retry is effectively worthless.